### PR TITLE
OS1: publish implement notebook page, changes to post page

### PR DESCRIPTION
**Pull request by ixv**
This implements the notebook page, and adds data fetching for notes and comment pagination on the notes page.

### CODE REVIEW

File: pkg/interface/publish/src/css/custom.css [hunk 1]
```diff
@@ -76,6 +76,20 @@ a {
...
+  background: rgba(42, 167, 121, 0.1);
+}
+
+.NotebookButton {
+  padding: 8px 12px;
+  border-radius:2px;
+  cursor: pointer;
+}
+
+.NotebookTab {
```
Comment (matildepark) [inline pkg/interface/publish/src/css/custom.css]: This can be `pv3 ph2` instead of making a custom class for this

**Comment by matildepark** [comment]
Screen Shot 2020-02-03 at 2 13 12 PM

Sorry, catching up here — This renders the `/settings` view always and the elements are disconnected. Intentional?

**Comment by ixv** [comment]
I think you need to pull

**Comment by matildepark** [comment]
Possibly — thought a checkout did that. One sec

**Comment by matildepark** [comment]
```
error in %publish
%load-failed
! nest-fail
! /~zod/home/0/app/publish/hoon:<[95 15].[95 33]>
! /~zod/home/0/app/publish/hoon:<[95 5].[230 7]>
! /~zod/home/0/app/publish/hoon:<[93 5].[230 7]>
! /~zod/home/0/app/publish/hoon:<[91 5].[230 7]>
! /~zod/home/0/app/publish/hoon:<[90 5].[230 7]>
!   /~zod/stage/~2020.1.25..00.29.51..86a9/sys/vane/gall
    <[1.297 13].[1.297 49]>
/~zod/stage/~2020.1.25..00.29.51..86a9/sys/vane/gall:<[1.297 13].[1.297 49]>
/~zod/home/0/app/publish/hoon:<[90 5].[230 7]>
/~zod/home/0/app/publish/hoon:<[91 5].[230 7]>
/~zod/home/0/app/publish/hoon:<[93 5].[230 7]>
/~zod/home/0/app/publish/hoon:<[95 5].[230 7]>
/~zod/home/0/app/publish/hoon:<[95 15].[95 33]>
nest-fail
```

Should I boot a fresh ship

**Comment by ixv** [comment]
Yes, thats crash is from changes to the app state.

**Comment by matildepark** [comment]
OK, needs navigation from note to notebook views but we can slate that as a QA task since the views work.

**Comment by vvisigoth** [comment]
merge pon
—
~poldec-tonteg
http://urbit.org

**Comment by vvisigoth** [comment]
*on
—
~poldec-tonteg
http://urbit.org

**Pull request merged**